projects/lib/src/lib/models/luigi.ts

```diff
@@ -190,6 +190,7 @@ export interface LuigiNode extends PortalLuigiNodeExtensions {
   context?: Record<string, any>;
   visibleForFeatureToggles?: string[];
   statusBadge?: LuigiStatusBadge;
+  isProjectLevelExtension?: boolean;
```

Comment:
is it a Luigi property? or a custom thing?

Reply:
and where it's used? or supposed to be used?

Reply:
Hi @gkrajniak , sorry for the late response, this one slipped by.

It's a custom thing and it's gonna be used in the portal, similar to the `statusBadge` property above it. We are implementing stuff for a Thread Modelling finding, which I'm not allowed to publicy disclose here, but you can find the related PR here: [Jukebox PR 1943](https://github.tools.sap/dxp/jukebox/pull/1943).

Reply:
I think we could move it to the PortalLuigiNodeExtensions as well. It basically serves the same purpose as similar flags such as `configurationMissing?`.

Reply:
hi guys, I think the flag `isProjectLevelExtension` should be in `jukebox`
why? in portal there is no notion of the project, so the below makes little sense in the `openmfp` 

`isProjectLevelExtension: extClass.scope.type.toUpperCase() === 'PROJECT',`

you should model it having in jukebox as an interface (if there is not already)

```
JukeboxLuigiNode extends LuigiNode {
 isProjectLevelExtension
}
```

and in this fashion we are able to very easy see and have the notion what is jukebox specific

Reply:
I absolutely see the point and I will give it a try. Might make sense to see if other flags should be moved there as well.

Reply:
I agree some jukebox specifics could be moved at some point along